scheduler/src/cook/scheduler/offer.clj

```diff
@@ -38,6 +38,9 @@
                           {"HOSTNAME" hostname}
                           compute-cluster
                           (assoc "COOK_MAX_TASKS_PER_HOST" (cc/max-tasks-per-host compute-cluster)
+                                 ; Note that this is dynamic and can change from moment to moment.
+                                 ; This could be a problem in the agent-attributes-cache..... However,
+                                 ; the rebalancer ignores this constraint, so its safe to cache this.
```

Comment:
```suggestion
                                 ; This could be a problem in the agent-attributes-cache. However,
                                 ; the rebalancer ignores this constraint, so it's safe to cache this.
```

Reply:
Thanks.